### PR TITLE
2021 year in review

### PR DESCRIPTION
**Pull request by jdebacker**
This PR adds a 2021 year in review post.  It addresses Issue #62

### CODE REVIEW

**Comment by jdebacker** [comment]
Screen Shot 2021-12-28 at 9 02 49 PM

File: _posts/2021-12-28-2021-year-in-review.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+![](https://pslmodels.org/imgs/PSL.svg)
+
+ ------
+
+As 2021 winds down, I wanted to take a few minutes to reflect on the Policy Simulation Library's efforts over the past year.
+With an amazing community of contributors, supporters, and users, PSL has been able to make a real impact in 2021.
+
+The library saw two new projects achieve "cataloged" status: Tax Foundation's [Capital Cost Recovery model](https://github.com/TaxFoundation/capital-cost-recovery) and the Federal Reserve Bank of New York's [DSGE.jl model](https://github.com/FRBNY-DSGE/DSGE.jl).
+Both models satisfy all the the [PSL criteria](http://pslmodels.org/Catalog/library_criteria.html) for transparency and repoducibility.
```
Comment (MaxGhenis) [inline _posts/2021-12-28-2021-year-in-review.md:18]:
```suggestion
Both models satisfy all the the [PSL criteria](http://pslmodels.org/Catalog/library_criteria.html) for transparency and reproducibility.
```

File: _posts/2021-12-28-2021-year-in-review.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+These datasets can then be used in microsimulation models, such as [Tax-Calculator](https://taxcalc.pslmodels.org) to study the impact of federal tax laws across the states.
+Matt Jensen and Don Boyd have published several pieces with these tools, including in [*State Tax Notes*](https://www.taxnotes.com/special-reports/tax-cuts-and-jobs-act/repealing-salt-cap-state-state-impacts/2021/10/21/7bbv3)
+
+[PSL Foundation](https://psl-foundation.org) became an official business entity in 2021.
+While still awaiting a letter of determination for 501(c)(3) status from the IRS, PSL Foundation was able to raise more than $25,000 in the last few months of 2021 to support open source policy analysis!
+
+PSL community members continued to interact several times each week in our public calls.
+The [PSL Shop](https://pslmodels.org/shop#!/) was launched in 2021 so that anyone can get themselves some PSL swag (with some of each purchase going back to the PSL Foundation to support the Library).
+In addition, PSL hosted 20 Demo Day presentations from 11 different presenters!
```
Comment (MaxGhenis) [inline _posts/2021-12-28-2021-year-in-review.md:36]:
```suggestion
In addition, PSL hosted 20 [Demo Day presentations](http://blog.pslmodels.org/categories/#demo-days) from 11 different presenters!
```

File: _posts/2021-12-28-2021-year-in-review.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+We are looking forward to a great 2022!
+We expect the Library to continue to grow, foresee many interesting and helpful Demo Days, and are planning a DC PSL Workshop for March 2022.
+We hope to see you around these or other events!
+
+Best wishes from PSL for a happy and healthy New Year!
+
+Resources:
+* [PSL Foundation](https://psl-foundation.org)
+* [PSL Twitter Feed](https://twitter.com/PSLmodels)
+* [PSL YouTube](https://www.youtube.com/channel/UCf7WWCuZHs_FFLjuBW4a4_Q)
```
Comment (MaxGhenis) [inline _posts/2021-12-28-2021-year-in-review.md:52]:
```suggestion
* [PSL YouTube channel](https://www.youtube.com/channel/UCf7WWCuZHs_FFLjuBW4a4_Q)
```

**Review by MaxGhenis — APPROVED** [review]
Thanks @jdebacker!

**Comment by jdebacker** [comment]
Thanks for the quick review @MaxGhenis!

**Pull request merged**